share/spack/win-compiler-wrapper/cl.hxx

```diff
@@ -0,0 +1,134 @@
+#include <iostream>
+#include <string>
+#include <vector>
+#include <map>
+#include <windows.h>
+#include <tchar.h>
+#include <stdio.h>
+#include <strsafe.h>
+
+#define BUFSIZE 4096
+
+typedef std::vector<std::string> SpackList;
```

Comment:
IMO `StrList` is a better abbreviation (more clearly indicates this is just for strings).